### ISSUE TITLE
[BUG] callback trace  `max_token`  throw KeyError Exception

### ISSUE THREAD

**Issue by happened** [Arize-ai/phoenix#2070]
**Describe the bug**

when i use llamaindex v0.9.28

use replicate api as the llm, the replicate class does not have `max_token` param
```python
# example from llamaindex website
import os
os.environ["REPLICATE_API_TOKEN"] = "xxxx"
from llama_index.llms import Replicate
llama2_7b_chat = "meta/llama-2-7b-chat:8e6975e5ed6174911a6ff3d60540dfd4844201974602551e10e9e87ab143d81e"
llm = Replicate(
    model=llama2_7b_chat,
    temperature=0.01,
    additional_kwargs={"top_p": 1, "max_new_tokens": 300},
)
```

then 
```python
from phoenix.trace.llama_index import (
    OpenInferenceTraceCallbackHandler,
)
callback_handler = OpenInferenceTraceCallbackHandler()
service_context = ServiceContext.from_defaults(
    llm=llm,
    embed_model=BgeLargeZhEmbeddings(instructor_model_name="/data2/models/bge-large-zh"),
    callback_manager=CallbackManager(handlers=[callback_handler]),
)
query_engine=index.as_query_engine(similarity_top_k=1)
print(query_engine.get_prompts())

response=query_engine.query(xxxx")
```

it will throw keyError

```bash
-Exception type: KeyError
-Exception message: 'max_tokens'
**************************************************
Traceback (most recent call last):
  File "/data/software/anaconda3/envs/duoli/lib/python3.9/site-packages/phoenix/utilities/error_handling.py", line 34, in wrapper
    return func(*args, **kwargs)
  File "/data/software/anaconda3/envs/duoli/lib/python3.9/site-packages/phoenix/trace/llama_index/callback.py", line 300, in on_event_start
    payload_to_semantic_attributes(event_type, payload),
  File "/data/software/anaconda3/envs/duoli/lib/python3.9/site-packages/phoenix/trace/llama_index/callback.py", line 221, in payload_to_semantic_attributes
    "max_tokens": serialized["max_tokens"],
KeyError: 'max_tokens'

```

maybe the callback.py should set default value if the key not exist

**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
 - OS: [e.g. MacOS]
 - Notebook Runtime [e.g. Jupyter, Colab]
 - Version [e.g. 0.0.1]

**Additional context**
Add any other context about the problem here (e.x. a link to a colab)

**Comment by RogerHYang** [comment]
some notes
1. SERIALIZED is a [dict](https://github.com/run-llama/llama_index/blob/742b2dc34d54abf14ceb8597cbb837491776425d/llama_index/llms/base.py#L97)
2. [max_tokens](https://github.com/run-llama/llama_index/blob/742b2dc34d54abf14ceb8597cbb837491776425d/llama_index/llms/openai.py#L86) is openai specific. So we can relax that.
3. But it's not easy to tell which to exclude. For example, do we exclude [prompt_key](https://github.com/run-llama/llama_index/blob/742b2dc34d54abf14ceb8597cbb837491776425d/llama_index/llms/replicate.py#L39) for replicate? We don't want to inadvertently include any sensitive data.

**Comment by happened** [comment]
However, the entire trace failed because the 'SERIALIZED' element lacked a key, which I believe is unsuitable . @RogerHYang

**Comment by mikeldking** [comment]
@happened thanks for the catch! This fix will be in the next release. Thanks again for trying out phoenix.

**Comment by RogerHYang** [comment]
@happened we just published a fix in v2.4.1. Let us know if you encounter more issues. Thanks!
